src/components/Form/FormFields/TextFormField.tsx

```diff
@@ -90,7 +90,7 @@ const TextFormField = forwardRef((props: TextFormFieldProps, ref) => {
         ref={ref as React.Ref<HTMLInputElement>}
         id={field.id}
         className={classNames(
-          "cui-input-base peer",
+          "cui-input-base peer pr-9",
```

Comment:
revert this change as it's unrelated to this pr
```suggestion
          "cui-input-base peer",
```